enable/tests/qt4/test_mouse_wheel.py

```diff
@@ -57,7 +56,7 @@ def test_vertical_mouse_wheel(self):
                 QtCore.Qt.NoModifier,
                 QtCore.Qt.Vertical,
             )
-        else:
+        elif is_qt5:
             qt_event = QtGui.QWheelEvent(
                 QtCore.QPoint(0, 0),
                 self.window.control.mapToGlobal(QtCore.QPoint(0, 0)),
```

Comment:
nit: should these be QPointF as well as opposed to just QPoint?

ref: https://doc.qt.io/qt-5/qwheelevent-obsolete.html

Reply:
Apparently PySide2 doesn't like this.